### ISSUE TITLE
Inject CLUSTER_NAME env variable for each NSM appended client

### ISSUE THREAD

**Issue by denis-tingaikin** [networkservicemesh/cmd-admission-webhook-k8s#163]
## Description

Currently `clientinfo` chain element can work with CLUSTER_NAME env https://github.com/networkservicemesh/sdk/blob/main/pkg/tools/clientinfo/clientinfo.go#L30

But we dont use it yet, because k8s at this moment doesn't have an option to get cluster name without any special complex steps.

## Solution

1. Wait when https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/2149-clusterid#property-idk8sio will be able for k8s users.
2. Add check in [cmd-admission-webhook-k8s](https://github.com/networkservicemesh/cmd-admission-webhook-k8s) on do we have a CRD related to [k8s about api](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/2149-clusterid#property-idk8sio )
3. Add CLUSTER_NAME env variable into NSC with value from cluster property crd.

**Comment by edwarnicke** [comment]
@denis-tingaikin https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/2149-clusterid#property-idk8sio is a CRD.  It is either already set when we run our admissions controller, in which case we simply use it's value, or it isn't in which case we should setup the CRD and set it to a value ourselves. 

Its not at all clear to me what the https://github.com/kubernetes-sigs/about-api controller would do, or when it would expect to be in GA.

**Comment by denis-tingaikin** [comment]
>@denis-tingaikin https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/2149-clusterid#property-idk8sio is a CRD. It is either already set when we run our admissions controller, in which case we simply use it's value, or it isn't in which case we should setup the CRD and set it to a value ourselves.

Yeah, that what we'll do when https://github.com/kubernetes-sigs/about-api  will be ready. 

Currently, as I can see CRD is not ready and is not using in k8s. Correct me if I wrong.

**Comment by edwarnicke** [comment]
@denis-tingaikin Yes... but what is the controller doing other than adding the CRD?  Is there any reason for us not to add the CRD if its not already present?
